FirebaseSwiftUI/FirebaseEmailAuthSwiftUI/Sources/FirebaseEmailAuthSwiftUI/Views/EmailAuthButton.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+public protocol FUIButtonProtocol: View {
+  var configuration: EmailAuthButtonConfiguration { get }
+}
+
+public class EmailAuthButtonConfiguration {
+  public var buttonText: String
```

Comment:
In SwiftUI, you configure views by using view modifiers. They allow for a much more scalable configuration of views.